config/clock.rb

```diff
@@ -24,6 +22,11 @@ class Clock
   every(1.hour, 'SendChaseEmailToProviders', at: '**:35') { SendChaseEmailToProvidersWorker.perform_async }
   every(1.hour, 'SendChaseEmailToCandidates', at: '**:40') { SendChaseEmailToCandidatesWorker.perform_async }
   every(1.hour, 'UpdateFeatureMetricsDashboard', at: '**:45') { UpdateFeatureMetricsDashboard.perform_async }
+  every(1.hour, 'SyncNewCycleIfRequired', at: '**:05') do
+    if Time.zone.now.between?(CycleTimetable.find_closes, CycleTimetable.find_closes + 1.hour)
```

Comment:
As mentioned in the PR description, it felt a bit strange making a separate worker to kick off here that does this check.